src/olympia/lib/crypto/tests/test_signing.py

```diff
@@ -147,12 +147,13 @@ def test_sign_file(self):
                 assert mozillasf.read().endswith(b'\n\n')
 
     def test_sign_file_non_ascii_filename(self):
-        src = self.file_.file_path
         # Pretend file on filesystem contains non-ascii characters. The
         # upload_to callback won't let us, so emulate what it does - as long as
         # we can read the file afterwards details don't matter.
-        self.file_.file.name = f'{self.file_.addon.pk}/wébextension.xpi.zip'
-        os.rename(src, self.file_.file_path)
+        old_file_path = self.file_.file_path
+        old_file_dirs = os.path.dirname(self.file_.file.name)
+        self.file_.file.name = f'{old_file_dirs}/wébextension.zip'
```

Comment:
I know it did it before, but `os.path.join` please 🙂

Reply:
Totally right. https://github.com/mozilla/addons-server/pull/19486/commits/e837160bb10b6cee6b5879bd4578b1425f81b67e